### ISSUE TITLE
Adding an onAnimationFinished listener

### ISSUE THREAD

**Issue by THMCombine** [krsticdragan/AutoTypingTextView#2]
Hello,

Would it be possible to add an onAnimationFinished listener? It would allow me to further animate other views based on when this is done typing.

**Comment by SaheelSiyam** [comment]
Yes, that would be very helpful. Possibly a method that finds out the last character typed?

Otherwise, this textview is nice and simple, and it is useful for my video game

**Comment by SaheelSiyam** [comment]
Or just go to
[https://github.com/iamporus/TypedTextView](url)
